libs/common-system/src/main/java/org/codice/ddf/configuration/SystemBaseUrl.java

```diff
@@ -240,13 +240,13 @@ public String getHttpsPort() {
   }
 
   /**
-   * Gets the current host name or IP address from the system properties, or {@link #DEFAULT_HOST}
-   * if not set.
+   * Gets the lowercase current hostname or IP address from the system properties, or {@link
+   * #DEFAULT_HOST} if not set.
    *
-   * @return host name, IP address or {@link #DEFAULT_HOST}
+   * @return hostname, IP address, or {@link #DEFAULT_HOST}
```

Comment:
This class is exported by this bundle. I'd be curious how this would affect down stream projects. Why didn't we change the comparison in the problem area to ignore case?

Reply:
I was wondering that as well. Is there a reason not to just do `equalsIgnoreCase` in the CSRF filter? W3 recommends assuming URLs are case-sensitive unless you know otherwise, like in the CSRF filter. https://www.w3.org/TR/WD-html40-970708/htmlweb.html

Reply:
❓ Shouldn't hostnames be normalized (lowercased) throughout our system? 
I realize this PR is meant to fix this specific bug, but I can't help but wonder if it's a sign we should have it normalized on the back end when we're passing it around.

Reply:
This change would normalize on the backend, with the exception of system property injection via blueprint/metatype.

Why can't the hostname just be set as lowercase in `system.properties`? Seems like something we could check and log a warning for.

Reply:
If we are using this method to display this value anywhere we are losing the original casing as well.

Reply:
Update added to include case insensitivity within the csrf filter. Since hostnames are case-insensitive, we're keeping the lowercase-format for SystemBaseUrl hostname methods update to keep hostnames consistent and normalized on the backend. Like @brendan-hofmann mentioned, if code needs to have the exact casing that's written in the system.properties file, they can access the property directly instead of asking the system what it's hostname is.